src/scene/sprite/Sprite.ts

```diff
@@ -409,4 +304,18 @@ export class Sprite extends Container implements View
             this._setHeight(convertedHeight, this._texture.orig.height);
         }
     }
+
+    public override onViewUpdate()
+    {
+        this._renderBoundsDirty = true;
+        super.onViewUpdate();
+    }
+
+    /**
+     * @deprecated since 8.2.5 (Use `renderBounds` instead)
+     */
+    get sourceBounds()
+    {
+        return this.renderBounds;
+    }
```

Comment:
```suggestion
    /**
     * @deprecated since 8.2.5 (Use `bounds` instead)
     */
    get sourceBounds()
    {
        return this.bounds;
    }
```